src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -343,20 +343,22 @@ private async Task SolutionManager_CacheUpdatedAsync(TimeSpan timeSpan, string e
 
         private async ValueTask RefreshWhenNotExecutingActionAsync(RefreshOperationSource source, TimeSpan timeSpanSinceLastRefresh)
         {
+            var sw = Stopwatch.StartNew();
```

Comment:
nit: sw probably on line 355.

Reply:
For now, I want to keep on 356, because it's still part of wait duration. 
After I make initial dashboard, we can tune up things to make more sense.